js/src/reports/programs/campaign-name-cell.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppTooltip from '.~/components/app-tooltip';
+import './campaign-name-cell.scss';
+
+/**
+ * Shows Tooltip {@link Notice}
+ * providing information about the conversion status of PMax campaigns
+ *
+ *
+ * @param {Object} props React props.
+ * @param {string} props.name Campaign Name
+ * @param {string} props.isConverted Weather the campaign is converted
+ * @return {JSX.Element} {@link Notice} element with the info message and the link to the documentation.
```

Comment:
❓ 
What is this `{@link Notice} ` for? 🤔

Reply:
Yeah `{@link Notice}` should not be there, it was there because I was working with the Notice component in another PR. Thanks for catching this. I have updated: [03e26db](https://github.com/woocommerce/google-listings-and-ads/pull/1496/commits/03e26db7b4228c7bf895f85105e5662830397777)